### DISCUSSION TITLE
Keyboard event for onNodeClick callback - what is the best way of implementing Ctrl + Click on a node?

### DISCUSSION THREAD

**Discussion by IRBorisov** [reaviz/reagraph#221] [Q&A]
Maybe Im missing something, but I cant see a way to get to ctrlKey state from inside of onNodeClick  without tracking KeyDown events from outside of canvas container.
Any ideas how to implement Ctrl + Click on node without external state?
Maybe I should open suggestion to expose mouse events in callbacks?

**Comment by amcdnl** [comment]
@IRBorisov - Happy to expose those events. Want to make a PR w/ them as the second argument?

**Reply by IRBorisov** [reply]
Sure I'd like to contribute. It is going to take some time though. Need to figure out this project build and PR process on github as it is going to be my first one
